docs/content/concepts/assets/software-defined-assets.mdx

```diff
@@ -67,15 +67,38 @@ By default, the name of the decorated function, `my_asset`, is used as the asset
 
 ### Assets with dependencies
 
-Software-defined assets can depend on other software-defined assets. In this section, we'll show you how to define:
+Software-defined assets can depend on other software-defined assets. In Dagster, you can either define dependencies that just specify materialization order, or you can define dependencies that also load the contents of the upstream asset. You can also define dependencies on external assets.
+
+In this section, we'll show you how to define:
 
 - [Basic asset dependencies](#defining-basic-dependencies)
-- [Explicit asset dependencies](#defining-explicit-dependencies)
+- [Basic data-loading asset dependencies](#defining-explicit-dependencies)
+- [Explicit data-loading asset dependencies](#defining-explicit-dependencies)
 - [External asset dependencies](#defining-external-asset-dependencies)
 
 #### Defining basic dependencies
 
-The easiest way to define an asset dependency is to include an upstream asset name as an argument to the decorated function.
+You can define a dependency between two assets by passing the upstream asset to the `deps` parameter in the downstream asset's `@asset` decorator.
+
+In this example, `upstream_asset` creates a new table (`sugary_cereals`) by selecting records from the `cereals` table. Then `downstream_asset` creates a new table (`shopping_list`) by selecting records from `sugary_cereals`
+
+```python file=/concepts/assets/non_argument_deps.py startafter=start_marker endbefore=end_marker
+from dagster import asset
+
+
+@asset
+def upstream_asset() -> None:
+    execute_query("CREATE TABLE sugary_cereals AS SELECT * FROM cereals")
+
+
+@asset(deps=[upstream_asset])
+def downstream_asset() -> None:
+    execute_query("CREATE TABLE shopping_list AS SELECT * FROM sugary_cereals")
+```
+
+#### Defining basic data-loading dependencies
```

Comment:
to me, i don't know what "managed-loading" means off the top of my head. we could definitely give it meaning, but then i wonder if people skimming through the concepts to find the answer to a question will miss this section because they don't know what the header means

Reply:
My concern about "data-loading" dependencies is that, in many cases, `deps` dependencies will involve loading data, it's just that it's happening inside the `@asset`-decorated function.

To anchor, do you have an example in mind of a question that they might be skimming to answer?

Reply:
yeah that's a good point, i hadn't considered that. 

I guess the question would be like "I've read the tutorial but forgot some syntax, how do i define the dependency where dagster loads the data for me?" - which i guess `managed-loading` would match up with that.

---

To do a little exercise:

I think if I were to write a super verbose heading, I'd end up with something like "defining dependencies where dagster will load the data from the upstream asset into memory for you".

Shortening that - "defining dependencies where dagster manages loading the upstream data" 

could be shorter still, but i'm not finding the right way to do that atm. i think bringing in "dagster manages" in the second version indicates that "managed-loading" might be where this ends up anyway. i'll change it and can always revisit later if needed